content/en/networks/spacenet/details/index.md

```diff
@@ -15,16 +15,18 @@ toc: true
 
 **Maintainer**: [ConsensusLab](https://consensuslab.world)
 
-Spacenet is not _just_ another Filecoin testnet. It has been modified to run the high-performance [Trantor BFT consensus](https://github.com/filecoin-project/mir/tree/main/pkg/systems/trantor), implemented atop [Mir](https://github.com/filecoin-project/mir), a framework for implementing distributed protocols. And did we forget to mention Spacenet comes with built-in [Interplanetary Consensus (IPC)]({{< relref "basics/interplanetary-consensus/overview" >}}) support?
+Spacenet is not _just_ another Filecoin testnet. It has been modified to run the high-performance [Trantor BFT consensus](https://github.com/filecoin-project/mir/tree/main/pkg/trantor), implemented atop [Mir](https://github.com/filecoin-project/mir), a framework for implementing distributed protocols. And did we forget to mention Spacenet comes with built-in [Interplanetary Consensus (IPC)]({{< relref "basics/interplanetary-consensus/overview" >}}) support?
```

Comment:
Link checker flagged https://github.com/filecoin-project/mir/tree/main/pkg/systems/trantor. Manual check confirmed file no longer exists. I found https://github.com/filecoin-project/mir/tree/main/pkg/trantor and edited the link accordingly. Can someone confirm that this is correct?

@johnnymatthews

Reply:
That new link is correct.